### ISSUE TITLE
Bug: R-series instances have a different device node for volume attachments

### ISSUE THREAD

**Issue by kdunn926** [quantstamp/eth-router#47]
Looks like [this](https://github.com/quantstamp/eth-router/commit/188c2312c7e51ea1394435a31a7180da11bac884) change is incompatible with the default `data_disk` var in the Ansible playbook:

```
TASK [blockchain : Install lvm2] ***************************************************************************
ok: [a.b.c.d]

TASK [blockchain : Create chaindata LVM volume group] ******************************************************
fatal: [a.b.c.d]: FAILED! => {"changed": false, "msg": "Device /dev/xvdg not found."}
```

**Comment by lazzarello** [comment]
Even weirder, downgrading back to `r5.xlarge` from `r5.2xlarge` succeeded without error.

**Comment by lazzarello** [comment]
I can't reproduce this in the `master` branch. I've upgraded and downgraded from `c4.large` to `r5.large` successfully each time. I think this might apply to a branch?

**Comment by kdunn926** [comment]
Strange indeed. Can you share how you're running `ansible-playbook` for a blockchain node once Terraform has provisioned the resources?

**Comment by lazzarello** [comment]
I figured it out (sort of). Provisioning a *new blockchain instance* will fail on the R series because the LVM tasks try and create a physical volume with the device that does not exist.

But the mystery began when I had an existing instance, created with the C class instance size and upgraded the same instance to an R series. The device name changed but `pvscan` didn't seem to care. It automatically picked that up and there was no data loss...

**Comment by lazzarello** [comment]
I think this could be fixed with documentation

**Comment by kdunn926** [comment]
Ah, if the variable override is passed as shown above then it should certainly work as-expected. In the [feature-multi-blockchains](#45 ) PR we did adjust the Ansible `hosts` file generation in `outputs.tf` to pass this variable in as an [Inventory-level override](https://github.com/quantstamp/eth-router/blob/master/deploy/terraform/outputs.tf#L17) - so maybe this issue is inadvertently fixed with #45 .
